test/cloudops.go

```diff
@@ -59,6 +60,12 @@ func compute(t *testing.T, driver cloudops.Ops) {
 	groupInfo, err := driver.InspectInstanceGroupForInstance(instanceID)
 	require.NoError(t, err, "failed to inspect instance group")
 	require.NotNil(t, groupInfo, "got nil instance group info from inspect")
+
+	err = driver.SetCountForInstanceGroup(instanceID, int64(3))
+	require.NoError(t, err, "failed to set node count for instance group")
+	// TODO: Once current node count is tracked in instanceGroupInfo,
```

Comment:
Add commented above, let's add the current node count support as this will be required by users of this API before invoking the set.